avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -85,6 +85,8 @@
   public static final String CONNECT_VERSION_PROP = "connect.version";
   public static final String CONNECT_DEFAULT_VALUE_PROP = "connect.default";
   public static final String CONNECT_PARAMETERS_PROP = "connect.parameters";
+  public static final String CONNECT_NAMED_MAP_PROP = "connect.named.map";
+  public static final String CONNECT_NAMED_MAP_PROP_TRUE = Boolean.toString(true);
```

Comment:
just `"true"`? or simply `true`? since prop values can be any json compatible object.